corehq/apps/export/forms.py

```diff
@@ -260,6 +261,13 @@ def get_mobile_user_and_group_slugs(self, data):
         return [n[1] for n in matches]
 
 
+location_query_help_text = ugettext_lazy(mark_safe(
```

Comment:
I looked into using a shared constant with
https://github.com/dimagi/commcare-hq/blob/e8ef5044c898d7972dadce86510a1a7b71605c96/corehq/apps/reports/filters/users.py#L219-L223
but we don't have a good place in commcare-hq for constants shared across the system.